service/mantle/shopify/ApiServices.xml

```diff
@@ -2001,10 +1945,32 @@
         <in-parameters>
             <parameter name="systemMessageId" required="true"/>
         </in-parameters>
+        <out-parameters>
+            <parameter name="paymentGatewayResponseId"/>
+        </out-parameters>
         <actions>
-            <service-call name="mantle.shopify.ApiServices.send#SystemMessageRest" 
+            <entity-find-one value-field="SM" entity-name="moqui.service.message.SystemMessage">
+                <field-map field-name="systemMessageId"/>
+            </entity-find-one>
+<!--            <if condition="SM.statusId.equals('sent')">-->
+<!--                <return/>-->
+<!--            </if>-->
+            <entity-find-one entity-name="mantle.account.payment.Payment" value-field="payment">
+                <field-map field-name="orderId" from="SM.orderId"/>
+            </entity-find-one>
+            <service-call name="mantle.shopify.ApiServices.send#SystemMessageRest"
                           in-map="[systemMessageId: systemMessageId]"
-                          out-map="context" />
+                          out-map="context"/>
+            <log message="\nSent refund message, about to update payment status\n"/>
+            <log message="\nResponse from the refund system message: ${response}\n"/>

```

Comment:
May want to comment out these logs for non development purposes. Especially when it's external data that might be sensitive.

Reply:
yea, thats legacy code, will certainly be removing all comments.